src/components/checkbox-group/daikin-checkbox-group.ts

```diff
@@ -0,0 +1,141 @@
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, unsafeCSS } from "lit";
+import {
+  customElement,
+  property,
+  queryAssignedElements,
+  state,
+} from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import tailwindStyles from "../../tailwind.css?inline";
+import type { MergeVariantProps } from "../../type-utils";
+import type { DaikinCheckbox } from "../checkbox/daikin-checkbox";
+import type { DaikinInputGroup } from "../input-group";
+
+const cvaCheckboxGroup = cva(["size-full", "flex"], {
+  variants: {
+    orientation: {
+      horizontal: ["flex-row", "flex-wrap", "gap-3"],
+      vertical: ["flex-col", "gap-2"],
+    },
+  },
+});
```

Comment:
The gap should be separated horizontally and vertically.

```suggestion
const cvaCheckboxGroup = cva(["size-full", "flex", "gap-x-3", "gap-y-2"], {
  variants: {
    orientation: {
      horizontal: ["flex-row", "flex-wrap"],
      vertical: ["flex-col"],
    },
  },
});
```